pxtblocks/fields/field_gridpicker.ts

```diff
@@ -648,4 +649,130 @@ export class FieldGridPicker extends Blockly.FieldDropdown implements FieldCusto
     private onClose_() {
         this.disposeTooltip();
     }
-}
\ No newline at end of file
+}
+
+Blockly.Css.register(`
+.blocklyGridPickerTooltip {
+    z-index: 995;
+}
+
+.blocklyGridPickerPadder {
+    outline: none;
+    box-shadow: 0px 0px 8px 1px rgba(0, 0, 0, .3)
+}
+
+.blocklyWidgetDiv .blocklyGridPickerRow {
+    display: table-row;
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu {
+    display: table;
+    outline: none;
+    border-spacing: 7px;
+}
+
+.blocklyGridPickerScroller {
+    outline: none;
+    padding: 4px;
+    border-radius: 4px;
+    position: relative;
+    -webkit-overflow-scrolling: touch;
+}
+
+.blocklyGridPickerPadder {
+    border-radius: 4px;
+    outline: none;
+    position: relative;
+}
+
+.blocklyGridPickerPadder .ui.input i.search.icon {
+    margin-top: -0.2rem;
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu .gridpicker-menuitem {
+    background: white;
+    cursor: pointer;
+    min-width: unset;
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu .gridpicker-menuitem-highlight, .blocklyWidgetDiv .blocklyGridPickerMenu .gridpicker-menuitem-hover {
+    background: #d6e9f8;
+    box-shadow: 0px 0px 0px 4px rgba(255, 255, 255, 0.2);
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu .gridpicker-option {
+    border: solid 1px black;
+    border-radius: 4px;
+    color: #fff;
+    font-size: 12pt;
+    font-weight: bold;
+    display: table-cell;
+    padding: 8px;
+    text-align: center;
+    vertical-align: top;
+    -webkit-user-select: none;
+    -moz-user-select: -moz-none;
+    -ms-user-select: none;
+        user-select: none;
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu .gridpicker-menuitem-content {
+    color: #fff;
+    font-size: 13px;
+    font-family: var(--pxt-page-font);
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu .floatLeft {
+    float: left;
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu .gridpicker-option.gridpicker-option-selected {
+    position: relative;
+}
+
+.blocklyWidgetDiv .blocklyGridPickerMenu .gridpicker-menuitem .gridpicker-menuitem-checkbox {
+    display: none;
+}
+
+.blocklyGridPickerTooltip {
+    z-index: 955;
+}
+
+.blocklyGridPickerSelectedBar {
+    display: flex;
+    padding-top: 5px;
+    justify-content: space-between;
+}
+
+.blocklyGridPickerSelectedImage {
+    padding: 3px;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.ui.input input.blocklyGridPickerSearchBar {
+    background: none;
+    border: none;
+    color: white;
```

Comment:
Should we be using theme colors here? (though I'd have to check if the css vars are in scope with the blockly css stuff...)

Reply:
Oh, just saw that this is copied from fieldeditor.less. I think it's fine to leave it as-is for now, then.